includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -122,14 +209,31 @@ public function register_routes() {
 	public function get_settings() {
 		return new WP_REST_Response(
 			[
-				'enabled_payment_method_ids'       => $this->gateway->get_upe_enabled_payment_method_ids(),
-				'available_payment_method_ids'     => $this->gateway->get_upe_available_payment_methods(),
-				'is_payment_request_enabled'       => 'yes' === $this->gateway->get_option( 'payment_request' ),
-				'is_test_mode_enabled'             => 'yes' === $this->gateway->get_option( 'testmode' ),
-				'payment_request_button_type'      => $this->gateway->get_option( 'payment_request_button_type' ),
-				'payment_request_button_theme'     => $this->gateway->get_option( 'payment_request_button_theme' ),
-				'payment_request_button_size'      => $this->gateway->get_option( 'payment_request_button_size' ),
-				'payment_request_button_locations' => $this->gateway->get_option( 'payment_request_button_locations' ),
+				/* Settings > General */
+				'is_stripe_enabled'                     => $this->gateway->is_enabled(),
+				'is_test_mode_enabled'                  => $this->gateway->is_in_test_mode(),
+
+				/* Settings > Payments accepted on checkout */
+				'enabled_payment_method_ids'            => $this->gateway->get_upe_enabled_payment_method_ids(),
+				'available_payment_method_ids'          => $this->gateway->get_upe_available_payment_methods(),
+
+				/* Settings > Express checkouts */
+				'is_payment_request_enabled'            => 'yes' === $this->gateway->get_option( 'payment_request' ),
+				'payment_request_button_type'           => $this->gateway->get_option( 'payment_request_button_type' ),
+				'payment_request_button_theme'          => $this->gateway->get_option( 'payment_request_button_theme' ),
+				'payment_request_button_size'           => $this->gateway->get_option( 'payment_request_button_size' ),
+				'payment_request_button_locations'      => $this->gateway->get_option( 'payment_request_button_locations' ),
+
+				/* Settings > Payments & transactions */
+				'is_manual_capture_enabled'             => 'no' === $this->gateway->get_option( 'capture' ),
```

Comment:
(minor) There's a `$WC_Stripe_UPE_Payment_Gateway->is_automatic_capture_enabled()` method on the UPE gateway - maybe in the future we can reuse that, by putting it in the `WC_Gateway_Stripe` class 👍